mne/baseline.py

```diff
@@ -143,42 +143,42 @@ def fun(d, m):
 
 
 def _check_baseline(baseline, times, sfreq, on_baseline_outside_data="raise"):
-    """Check if the baseline is valid, and adjust it if requested.
+    """Check if the baseline is valid and adjust it if requested.
 
-    ``None`` values inside the baseline parameter will be replaced with
-    ``times[0]`` and ``times[-1]``.
+    ``None`` values inside ``baseline`` will be replaced with ``times[0]`` and
+    ``times[-1]``.
 
     Parameters
     ----------
-    baseline : tuple | None
+    baseline : array-like | None
```

Comment:
can we specify the length of this array in the docstring? (it's an actual question, I don't know if we can)

Reply:
We do that to specify the shape, e.g. `array-like, shape (2,)` – would that work?

Reply:
Would work for me